.gitignore

```diff
@@ -0,0 +1,30 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+
+# IDE files
+.idea
+
+# The _build directory (remove the comment below to include it)
+# _build/
+
+# Files generated by popular Go packages (remove the comment below to include it)
+# node_modules/
+# *.log
+
+# Don't ignore .gitignore and .gitattributes files
+!.gitignore
+!.gitattributes
+demo
```

Comment:
Please add a newline at the end of the file, including the following test files.